src/pages/portfolio/Overview.tsx

```diff
@@ -29,6 +32,12 @@ export const Overview = () => {
   const shouldRenderTriggers = useSelector(calculateShouldRenderTriggersInPositionsTable);
   const shouldRenderActions = useSelector(calculateShouldRenderActionsInPositionsTable(true));
 
+  const handleViewUnopenedIsolatedOrders = useCallback(() => {
+    navigate(`${AppRoute.Portfolio}/${PortfolioRoute.Orders}`, {
```

Comment:
can't go to trade page because horizontal bar state is local to the component 
can't go to specific market in orders because that functionality doesn't exist